source/overview/license-and-subscription.rst

```diff
@@ -71,13 +71,31 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your Mattermost license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
 
-To renew your license in the meantime, contact your Customer Success Manager to start the renewal process. Alternatively, `contact us <https://mattermost.com/contact-us/>`_ for assistance.
+If you are a reseller, have a non-standard contract, or want to adjust the number of active users on your license please contact sales@mattermost.com.
```

Comment:
Question: Is there value in making this email address clickable?

Reply:
I don't see why not. :)